js/js.js

```diff
@@ -1,52 +1,96 @@
-$(document).ready(function() {
+function scrollTo(element, to, duration) {
+  var start = element.scrollTop,
+      change = to - start,
+      currentTime = 0,
+      increment = 20;
 
-  function appendLink(id) {
-    $(id).next('div').find('.link').empty();
-    $(id).next('div').find('.link').append("<small>Link to this command: <a href='https://amiaopensource.github.io/ffmprovisr/index.html" + id + "'>https://amiaopensource.github.io/ffmprovisr/index.html" + id + "</a></small>");
+  var animateScroll = function(){        
+    currentTime += increment;
+    var val = Math.easeInOutQuad(currentTime, start, change, duration);
+    element.scrollTop = val;
+    if(currentTime < duration) {
+        setTimeout(animateScroll, increment);
+    }
+  }
+  animateScroll()
+}
+
+//t = current time, b = start value, c = change in value, d = duration
+Math.easeInOutQuad = function (t, b, c, d) {
+    t /= d/2;
+    if (t < 1) return c/2*t*t + b;
+    t--;
+    return -c/2 * (t*(t-2) - 1) + b;
+}
+
+function appendLink(id) {
+  let link = document.getElementById(id).nextElementSibling.querySelector('.link')
+  if (link) {
+    link.innerHTML = ("<small>Link to this command: <a href='https://amiaopensource.github.io/ffmprovisr/index.html#" + id + "'>https://amiaopensource.github.io/ffmprovisr/index.html#" + id + "</a></small>")
   }
+}
 
-  function moveToRecipe(id) {
-    document.getElementById(id.substring(1)).checked = true;
-    $('html, body').animate({ scrollTop: $(id).offset().top }, 1000);
+function moveToRecipe(id) {
+  document.getElementById(id).checked = true;
+  scrollTo(document.body, 0, 1000); 
+  appendLink(id)
+}
+
+// open recipe window if a hash is found in URL
+if (window.location.hash) {
+  id = window.location.hash.slice(1)
+  moveToRecipe(id)
+}
+
+// add hash URL when recipe is opened
+let recipes = document.querySelectorAll('label[class="recipe"]')
```

Comment:
(minor) `recipes` (and some of the other `let` variables) can be `const`s, no?

Reply:
yes! thanks @kfrn -- clearly was just being lazy here